src/shapes/Object/types/index.ts

```diff
@@ -1,7 +1,7 @@
 import { FabricObjectProps } from './FabricObjectProps';
 
-export { SerializedObjectProps } from './SerializedObjectProps';
-export { FabricObjectProps };
+export { type SerializedObjectProps } from './SerializedObjectProps';
+export { type FabricObjectProps };
```

Comment:
we are adding a lint rule that *SHOULD* catch this, but for now it doesn't work for some reason.
It catches imports but not exports yet.
https://github.com/fabricjs/fabric.js/pull/8907

Reply:
@asturur 
Thanks for the explanation.

This PR will be closed!